docs/UserGuide.md

```diff
@@ -81,7 +81,7 @@ The `Chart Panel` (an example is shown above) contains _Ramp Charts_ and _Contri
 
 ### Code Panel
 
-The `Code Panel` allows users to see the code attributed to a specific author. Click on the name of the author in the `Chart Panel` to display the `Code Panel` on the right.
+The `Code Panel` allows users to see the code attributed to a specific author. Click on the `</>` icon beside the name of the author in the `Chart Panel` to display the `Code Panel` on the right.
```

Comment:
I think it would be better if we could include the actual icon instead of `</>`.

Reply:
Same as above; I can't find any icons that is close enough to display that icon unless if we import third party libraries.